src/client/QXmppAtmManager.cpp

```diff
@@ -61,6 +61,7 @@ using namespace QXmpp::Private;
 /// \param trustStorage trust storage implementation
 ///
 QXmppAtmManager::QXmppAtmManager(QXmppAtmTrustStorage *trustStorage)
+    : QXmppTrustManager(trustStorage)
 {
     m_trustStorage = trustStorage;
```

Comment:
Yes it is needed because that is the ATM specific trust storage (`QXmppAtmTrustStorage`) which is only needed by `QXmppAtmManager`. `QXmppOmemoManager` needs only `QXmppTrustStorage` and not `QXmppAtmTrustStorage`.